packages/backend/src/asset/errors.ts

```diff
@@ -1,8 +1,25 @@
 export enum AssetError {
   DuplicateAsset = 'DuplicateAsset',
-  UnknownAsset = 'UnknownAsset'
+  UnknownAsset = 'UnknownAsset',
+  InuseAsset = 'InuseAsset'
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
 export const isAssetError = (o: any): o is AssetError =>
   Object.values(AssetError).includes(o)
+
+export const errorToCode: {
+  [key in AssetError]: number
+} = {
+  [AssetError.UnknownAsset]: 404,
+  [AssetError.DuplicateAsset]: 400,
+  [AssetError.InuseAsset]: 400
+}
+
+export const errorToMessage: {
+  [key in AssetError]: string
+} = {
+  [AssetError.UnknownAsset]: 'unknown asset',
+  [AssetError.DuplicateAsset]: 'Asset already exists',
+  [AssetError.InuseAsset]: 'Asset in use'
```

Comment:
When reading, 'Asset in use' it doesn't seem like an error to me right away. Maybe this can include something about the fact that you couldn't delete it?